.github/workflows/ci.yml

```
@@ -73,32 +33,76 @@ jobs:
           - omnibus
           - python
           - terraform
+    env:
+      DOCKER_BUILDKIT: 1
+      DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
+      DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
     steps:
-      - name: Pull image from packages
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Prepare environment variables
         run: |
-          docker login docker.pkg.github.com -u x -p ${{ secrets.GITHUB_TOKEN }}
-          docker pull "docker.pkg.github.com/${{ github.repository }}/dependabot-core-ci:${{ github.sha }}"
-          docker logout
-      - name: Run Rubocop linting
+          BRANCH_REF="$(echo '${{ github.ref }}' | sed -E 's/[^A-Za-z0-9]+/-/g')"
+          echo ::set-env name=BRANCH_REF::$(BRANCH_REF)
```

```suggestion
          echo ::set-env name=BRANCH_REF::$(BRANCH_REF)
          echo ::set-env name=BASE_IMAGE::ubuntu:18.04
```

Idea: to de-dupe the later usages.
Maybe even `$(grep FROM Dockerfile | sed '...')`? Meh.